external/webserver/webserver_test_guide.md

```diff
@@ -0,0 +1,70 @@
+# Python Client to stress test Web Server
+
+### Features
+
+- Protocols supported: HTTPS & HTTP
+- HTTP Commands
+    - GET
+    - PUT
+    - POST
+    - DELETE
+- HTTP Header
+    - Supports Keep-alive
+    - Chunked Transfer
+    - Trailer Header
+
+## Prerequisites
+
+### Python3
+**Python modules**
+
+- Requests
+
+` pip install requests`
+
+- Prettytable
+
+` pip install prettytable`
+
+### Source Code
+https://github.com/Samsung/TizenRT
+Branch: master
+
+### Certificate for HTTPS protocol
+**Certificate Generation**
+1. Open san.cnf in a text editor.
```

Comment:
san.cnf file not included, write sample data

Reply:
added san.cnf content